src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -129,47 +118,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         HttpServletResponse.SC_METHOD_NOT_ALLOWED, "Unimplemented: " + request.getRequestURI());
   }
 
-  /**
-   * Retrieves a user in the database by id, or null if said id does not exist.
-   *
-   * @param id the user's id
-   * @return the user with id's data or null.
-   */
-  private User getUserById(String id) {
-    // MOCK IMPLEMENTATION
-    if (id.equals("current")) {
-      return USER_MAP.get(CURRENT_USER_KEY);
-    }
-    return USER_MAP.get(id);
+  /** Getters and Setters for connected objects. */
+  public Datastore getDatastore() {
```

Comment:
Why create getters and setters for the datastore? AFAICT they are never used.

Reply:
Will remove `[get|set]Datastore`.